atomicapp/nulecule/base.py

```diff
@@ -88,11 +89,16 @@ def __init__(self, id, specversion, graph, basepath, metadata=None,
         self.metadata = metadata or {}
         self.graph = graph
         self.requirements = requirements
-        self.config = config or {}
+        if isinstance(config, Config):
+            self.config = config
+        else:
+            self.config = Config(namespace=namespace, answers=config,
+                                 cli=cli)
```

Comment:
Bit confused as to why there is an if statement here, if we are refactoring config, should we only pass it in once?